handlers.go

```diff
@@ -223,12 +206,12 @@ func mkVmap(keys []string, values []interface{}) (*map[string]interface{}, error
 		// convert from sql.RawBytes to string
 		rbp, ok := values[i].(*sql.RawBytes)
 		if !ok {
-			return &ret, fmt.Errorf("sql conversion error")
+			return ret, fmt.Errorf("sql conversion error")
 		}
 		s := string(*rbp)
 		ret[keys[i]] = interface{}(&s)
 	}
-	return &ret, nil
+	return ret, nil
 }
 
 // mkSQLRow() returns a list of interface{} of the given length,
```

Comment:
Yes you're right, it would not compile with []*sql.RawBytes.
I would suggest:
```
vals = make([]sql.RawBytes, len(cols))
valPointers = make([]interface{}, len(cols))
for i, _ := range vals {
    valPointers[i] = &vals[i]
}
err := rows.Scan(valPointers...)
```

So your `mkVmap` will be `func mkVmap(keys []string, values []sql.RawBytes)`. No need for converting, will be concise.